src/Storages/ExternalStream/Kafka/Kafka.cpp

```diff
@@ -524,6 +527,16 @@ int Kafka::onStats(struct rd_kafka_s * rk, char * json, size_t json_len, void *
     return 0;
 }
 
+void Kafka::onLog(const struct rd_kafka_s * rk, int level, const char * fac, const char * buf)
+{
+    if (level < 4)
+        LOG_ERROR(cbLogger(), "{}|{} buf={}", rd_kafka_name(rk), fac, buf);
+    else if (level == 4)
+        LOG_WARNING(cbLogger(), "{}|{} buf={}", rd_kafka_name(rk), fac, buf);
+    else
+        LOG_INFO(cbLogger(), "{}|{} buf={}", rd_kafka_name(rk), fac, buf);
```

Comment:
In normal cases, there won't be too much logs. If `debug` is enabled with the `properties` setting, there will be a lot more logs. But that's also the point fo adding this, because we want such logs to help us for debugging.